client/ayon_core/plugins/publish/extract_otio_review.py

```diff
@@ -62,6 +61,8 @@ def process(self, instance):
             make_sequence_collection
         )
 
+        self.temp_file_head = self._get_unique_file_prefix(instance)
```

Comment:
Honestly - we shouldn't really be setting this on the class instance because the plugin instance is shared between instances. Sure, we're always resetting it here - but it may produce odd side effects if someone else doesn't know about that and starts tweaking the code for this?

Reply:
That would need much deeper refactoring as there are many other occurrences of other variables.